src/views/Correction.vue

```diff
@@ -83,28 +83,32 @@
               <h1 id="correction-header">
                 Correction &mdash; {{ title }}
               </h1>
-              <p class="text-black">
-                Original Filing Date: {{ originalFilingDate }}
+              <p class="subtitle">
+                Correction for Record Conversion, filed on {{ originalFilingDate }}
```

Comment:
There are corrections for different filings. So you should replace `Record Conversion` with its corresponding type. 
For example, the following subtitle should be "Correction for Dissolution"
![image](https://github.com/bcgov/business-filings-ui/assets/60866283/ce6adc24-b5a9-4482-b11d-b566070d7a45)

Reply:
is there a type i can use for this?

Reply:
Haha, I see there was already a variable you could use... nice :)